src/Api/IriConverterInterface.php

```diff
@@ -26,12 +26,13 @@
      *
      * @param string $iri
      * @param bool   $fetchData
+     * @param array  $context
      *
      * @throws InvalidArgumentException
      *
      * @return object
      */
-    public function getItemFromIri(string $iri, bool $fetchData = false);
+    public function getItemFromIri(string $iri, bool $fetchData = false, array $context = []);
```

Comment:
What do you think about moving `fetchData` to a key of the context? It's really an implementation detail linked to Doctrine, it doesn't apply in most other contexts. Doing so will made the API nicer.